scaaml_intro/train.py

```diff
@@ -31,8 +31,9 @@
 
 def train_model(config):
     tf_cap_memory()
-    train_glob = f"datasets/{config['algorithm']}/train/*"
-    test_glob = f"datasets/{config['algorithm']}/test/*"
+    algorithm = config['algorithm']
```

Comment:
surprised pylint doesn't complain about not using double quotes here.

Reply:
It was, but not anymore. PTAL